climakitae/data/variable_descriptions.csv

```diff
@@ -0,0 +1,22 @@
+name,description,extended_description
+T2,2m Air Temperature,"Temperature of the air 2 meters above Earth's surface. This is the measure of air temperature used for most modeling applications."

```

Comment:
Suggest rename Air Temperature at 2m for consistency

Reply:
changed!